src/scenes/ProgressReports/ProgressReportRouter.tsx

```diff
@@ -2,9 +2,11 @@ import { Route, Routes } from 'react-router-dom';
 import { ChangesetContext } from '../../components/Changeset';
 import { NotFoundRoute } from '../../components/Error';
 import { ProgressReportDetail as ProgressReport } from './Detail';
+import { StatusHistory } from './StatusHistory';
 
 export const ProgressReportRouter = () => (
   <Routes>
+    <Route path=":reportId/history" element={<StatusHistory />} />
```

Comment:
"history" is pretty ambiguous. I'd go with "workflow-events" or "workflow-history"